### DISCUSSION TITLE
Configfiles with multiple Keys

### DISCUSSION THREAD

**Discussion by lgoette** [nix-community/plasma-manager#157] [Q&A] [answered]
Hello,
i want to change the Wallpaper of the Lockscreen and it seems like Configfiles with multiple Keys like
```
[Greeter][Wallpaper][org.kde.image][General]
Image=<image-path>
``` 
are not supoorted.
`"kscreenlockerrc"."Greeter"."Wallpaper"."org.kde.image"."General"."Image".value = <image-path>;` did not work.

Is there another way to adjust for that?

**Comment by magnouvean** [comment, marked as answer]
It is supported, simply use "/" as a separator:
```nix
kscreenlockerrc."Greeter/Wallpaper/org.kde.image/General".Image.value = <image-path>;
```

With the latest version you don't even need the `.value` so you can even use:
```nix
kscreenlockerrc."Greeter/Wallpaper/org.kde.image/General".Image = <image-path>;
```

**Reply by lgoette** [reply]
Oh thats great, thank you. :D
Is there actually some documentation? I don't want to bother with stupid questions :)

**Reply by lgoette** [reply]
I just saw that I could take this Information from the rc2nix output :)

**Reply by magnouvean** [reply]
We probably should add en example in the example configs though as it doesn't seem to be represented there.
